atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -12,11 +12,9 @@ Atlas supports Node.js applications using popular frameworks like [Faust.js](htt
 
 ### 1. Prepare your app for deployment
 
-First make sure that you've pushed your application's code to your GitHub repository. If you're not familiar with pushing code to a GitHub repository, see this guide: [Adding an existing project to GitHub using the command line](https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-an-existing-project-to-github-using-the-command-line).
+When building the front-end app, WPEngine will use ```npm run build``` by default. If you want to specify a build command, you can do so in one of two ways:
 
-Your application needs to include a `wpe-build` script in the [`scripts`](https://docs.npmjs.com/cli/v8/using-npm/scripts) field in your project's `package.json` file. It should execute the build command for your project. This is the script that the Atlas platform will execute when it builds your app.
-
-For example, the `package.json` file for a Faust.js or Next.js app would need to be updated as shown below.
+1. adding a "wpe-build" script in the package.json file for your app.
```

Comment:
@dgallardox  I think we should capitalize the first letter here, remove the "ing" ending from "Adding", and add backticks to these inline code/file references, like this:
1. Add a `wpe-build` script in the `package.json` file for your app.